docs/source/quickinstall.rst

```diff
@@ -8,39 +8,52 @@ This quick install guide outlines the basic steps needed to install OpenMC on
 your computer. For more detailed instructions on configuring and installing
 OpenMC, see :ref:`usersguide_install` in the User's Manual.
 
-----------------------------------------
-Installing on Linux/Mac with conda-forge
-----------------------------------------
-
-`Conda <https://conda.io/en/latest/>`_ is an open source package management
-system and environment management system for installing multiple versions of
-software packages and their dependencies and switching easily between them. If
-you have `conda` installed on your system, OpenMC can be installed via the
+--------------------------------------------------
+Installing on Linux/Mac with Mamba and conda-forge
+--------------------------------------------------
+
+`Conda <https://conda.io/en/latest/>`_ and `Mamba <https://mamba.readthedocs.io/en/latest/>`_
+are an open source package management systems and environments management
+system for installing multiple versions of software packages and their
+dependencies and switching easily between them. If you have `conda` installed
+on your system, OpenMC can be installed via the
 `conda-forge` channel. First, add the `conda-forge` channel with:
 
 .. code-block:: sh
 
     conda config --add channels conda-forge
 
+Then install `mamba`, which will later be used to install OpenMC in the conda environment.
+
+.. code-block:: sh
+
+    conda install mamba
+
 To list the versions of OpenMC that are available on the `conda-forge` channel,
 in your terminal window or an Anaconda Prompt run:
 
 .. code-block:: sh
 
-    conda search openmc
+    mamba search openmc
 
-OpenMC can then be installed with:
+First create and activate a new conda enviroment called `openmc-env` in which to install OpenMC.
 
 .. code-block:: sh
 
-    conda create -n openmc-env openmc
+    conda create -n openmc-env
+    conda activate openmc-env
 
-This will install OpenMC in a conda environment called `openmc-env`. To activate
-the environment, run:
+OpenMC can then be installed with:
 
 .. code-block:: sh
 
-    conda activate openmc-env
+    mamba install openmc
+
+You are now in a conda environment called `openmc-env` that has OpenMC installed.
+
+.. note:: If you are already familiar with conda for package management,
+          please note that OpenMC is currently only supported to be installed
+          with `mamba` and not `conda` (ie, `conda install openmc` may not work).
```

Comment:
I would just remove this note. It's not that we don't support conda, but just that conda is broken. However, once it's fixed (:crossed_fingers:) this documentation would become out of date.